pkg/cluster/cluster.go

```diff
@@ -212,13 +212,17 @@ func (c *Context) provisionControlPlane(
 
 	// create kubeadm config file
 	kubeadmConfig, err := c.createKubeadmConfig(
```

Comment:
We ignored this error previously, right?

Reply:
yes, that was bad